packages/web-components/fast-router/api-extractor.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+
```

Comment:
unlikely that it causes issues but these spaces seem suspect

Reply:
Shouldn't be a technical issue but it is interesting that the space is there.